contrib/custom-node-e2e/README.md

```diff
@@ -0,0 +1,23 @@
+# Node e2e test with custom CRI-O builds
+
+The behaviour of [Kubernetes node e2e tests](https://github.com/kubernetes/community/blob/41289f0f7e1fdc559c24e09c8e66c1f805fdfd52/contributors/devel/sig-node/e2e-node-tests.md) that use [CRI-O as the CRI](https://github.com/kubernetes/test-infra/tree/859a94267c109ec709326f60cadf5add07e87cf4/jobs/e2e_node/crio) which run on k8s CI could be replicated locally through GCP VM(s).
+
+This could be helpful to debug node e2e test failures that would come up during developing CRI-O itself. The script builds CRI-O on the local system, creates an installation bundle tarball and uploads it to a GCS bucket of user's choice. While running the tests, a [GCP instance is created](https://github.com/kubernetes/community/blob/41289f0f7e1fdc559c24e09c8e66c1f805fdfd52/contributors/devel/sig-node/e2e-node-tests.md#remotely) and the [Fedora CoreOS machine](https://github.com/kubernetes/test-infra/tree/859a94267c109ec709326f60cadf5add07e87cf4/jobs/e2e_node/crio/latest) consumes the output ignition config which installs the custom CRI-O build and e2e tests are run against it.
```

Comment:
```suggestion
This could be helpful to debug node e2e test failures that would come up during developing CRI-O itself. The create-ignition-config.sh script builds CRI-O on the local system, creates an installation bundle tarball and uploads it to a GCS bucket of user's choice. While running the tests, a [GCP instance is created](https://github.com/kubernetes/community/blob/41289f0f7e1fdc559c24e09c8e66c1f805fdfd52/contributors/devel/sig-node/e2e-node-tests.md#remotely) and the [Fedora CoreOS machine](https://github.com/kubernetes/test-infra/tree/859a94267c109ec709326f60cadf5add07e87cf4/jobs/e2e_node/crio/latest) consumes the output ignition config which installs the custom CRI-O build and e2e tests are run against it.
```

Reply:
Thanks, updated it.